hdfs_datanode/tox.ini

```
@@ -23,6 +23,10 @@ passenv =
 setenv =
     2.7.1: HDFS_RAW_VERSION=2.7.1
     2.7.1: HDFS_IMAGE_TAG=1.1.0-hadoop2.7.1-java8
+    DD_TRACE_DEBUG=true
+    DD_ENV=staging
+    DD_SERVICE=datadog-integrations-core
+    DD_TAGS="team:agent-integrations"
 commands =
     pip install -r requirements.in
-    pytest -v {posargs}
+    pytest --ddtrace -v {posargs}
```

```suggestion
    pytest -v {posargs}
```
it is better to rely on `PYTEST_ADDOPTS` defined in the pipeline